src/com/projectkorra/projectkorra/board/BendingBoardInstance.java

```diff
@@ -0,0 +1,112 @@
+package com.projectkorra.projectkorra.board;
+
+import com.projectkorra.projectkorra.BendingPlayer;
+import com.projectkorra.projectkorra.ability.CoreAbility;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.entity.Player;
+import org.bukkit.scoreboard.DisplaySlot;
+import org.bukkit.scoreboard.Objective;
+import org.bukkit.scoreboard.Scoreboard;
+
+import java.util.*;
```

Comment:
We try to avoid using wildcard imports. Is that something you'd be willing to change?